sdk/formrecognizer/azure-ai-formrecognizer/azure/ai/formrecognizer/_polling.py

```diff
@@ -44,7 +45,39 @@ def parse_operation_id(location):
     return operation_id
 
 
-class DocumentModelAdministrationLROPoller(LROPoller[PollingReturnType]):
+class DocumentModelAdministrationLROPoller(Protocol[PollingReturnType]):
+    """Implements a protocol followed by returned poller objects."""
+
+    @property
+    def details(self) -> Mapping[str, Any]: # pylint: disable=no-self-use, unused-argument
+        ...
+
+    def polling_method(self) -> PollingMethod[PollingReturnType]:  # pylint: disable=no-self-use
+        ...
+
+    def continuation_token(self) -> str:  # pylint: disable=no-self-use
+        ...
+
+    def status(self) -> str:  # pylint: disable=no-self-use
+        ...
+
+    def result(self, timeout: Optional[int] = None) -> PollingReturnType: # pylint: disable=no-self-use, unused-argument
+        ...
+
+    def wait(self, timeout: Optional[float] = None) -> None:  # pylint: disable=no-self-use, unused-argument
+        ...
+
+    def done(self) -> bool:  # pylint: disable=no-self-use
+        ...
+
+    def add_done_callback(self, func: Callable) -> None:  # pylint: disable=no-self-use, unused-argument
+        ...
+
+    def remove_done_callback(self, func: Callable) -> None:  # pylint: disable=no-self-use, unused-argument
+        ...
+
+
+class DocumentModelAdministrationClientLROPoller(LROPoller[PollingReturnType]):
```

Comment:
Left the longer name for the types we dont expose...although I am open to shorter name suggestions :)